### PR TITLE
Cleanup/startup args

### PR DESCRIPTION
**Pull request by renbauer-fnal**
Code cleanup associated with the startup arg feature - rebased out of feature/startup_args branch

### CODE REVIEW

File: src/library_manager.py [hunk 1]
```diff
@@ -77,22 +77,25 @@
 def get_storage_group(dict):
     sg = dict.get('storage_group', None)
     if not sg:
-	vf = dict.get('volume_family', None)
-	if vf:
-	    sg = volume_family.extract_storage_group(vf)
+        vf = dict.get('volume_family', None)
```
Comment (DmitryLitvintsev) [inline src/library_manager.py:80]: Are those tabs ? I believe they need to change to white spaces.

File: src/library_manager.py [hunk 2]
```diff
@@ -1077,31 +1089,31 @@ def __send_regret(self, ticket):
...
-		#The TCP handshake is in progress.
-		elif detail[0] == errno.EINPROGRESS:
-		    pass
-		else:
-		    Trace.log(e_errors.ERROR, "error connecting to %s (%s)" %
-			      (ticket['callback_addr'], os.strerror(detail)))
-
-            callback.write_tcp_obj(control_socket,ticket)
+            Trace.trace(self.trace_level + 10, "connecting to %s" % (ticket['callback_addr'],))
+            try:
```
Comment (DmitryLitvintsev) [inline src/library_manager.py:1095]: These seem to be tabs as well. Replace with white spaces.

File: src/enstore_start.py [hunk 3]
```diff
@@ -292,19 +302,20 @@ def check_user():
                       "but the enstore user is not found."
                 sys.exit(1)
 
-        #Extract the user name.
-	try:
+        # Extract the user name.
+        try:
```
Comment (DmitryLitvintsev) [inline src/enstore_start.py:306]: seem like tabs , replace with white spaces.

File: src/enstore_start.py [hunk 4]
```diff
@@ -606,182 +619,148 @@ def should_start(self, server_name):
...
-                     option.VALUE_USAGE:option.IGNORED,
-                     option.DEFAULT_NAME:"all",
-                     option.DEFAULT_TYPE:option.INTEGER,
-		     option.USER_LEVEL:option.ADMIN,
+        option.JUST: {option.HELP_STRING: "specify single server",
+                      option.VALUE_NAME: "just",
+                      option.VALUE_USAGE: option.REQUIRED,
+                      option.VALUE_TYPE: option.STRING,
+                      option.VALUE_LABEL: "server name",
+                      option.USER_LEVEL: option.ADMIN,
```
Comment (DmitryLitvintsev) [inline src/enstore_start.py:630]: tabs

File: src/enstore_start.py [hunk 5]
```diff
@@ -606,182 +619,148 @@ def should_start(self, server_name):
...
+                      },
+        option.ALL: {option.HELP_STRING: "specify all servers",
+                     option.VALUE_USAGE: option.IGNORED,
+                     option.DEFAULT_NAME: "all",
+                     option.DEFAULT_TYPE: option.INTEGER,
+                     option.USER_LEVEL: option.ADMIN,
```
Comment (DmitryLitvintsev) [inline src/enstore_start.py:636]: tabs

File: src/library_manager.py (hunk 1 above)
Comment (renbauer-fnal) [reply src/library_manager.py:80]: All of these instances look like white spaces to me in vim and pycharm. I'm not sure if it is an editor quirk but deleting and replacing with spaces gives me no change. Can you double check?

Thanks for the close look

Comment (DmitryLitvintsev) [reply src/library_manager.py:80]: OK, let's assume these are not tabs. Not that super important

**Review by DmitryLitvintsev — APPROVED** [review]

**Comment by renbauer-fnal** [comment]
Thanks Dmitry!

**Pull request merged**